app/Http/Controllers/Frontend/DevController.php

```diff
@@ -3,14 +3,16 @@
 namespace App\Http\Controllers\Frontend;
 
 use App\Http\Controllers\Controller;
+use App\Repositories\OAuthClientRepository;
 use Illuminate\Http\RedirectResponse;
 use Illuminate\Http\Request;
 use Illuminate\View\View;
 use Laravel\Passport\ClientRepository;
 
 class DevController extends Controller
 {
-    public function renderAppList(): View {
+    public function renderAppList(): View
```

Comment:
sure, i'll do that. Besides the braces, the formatting will mismatch i think, because, i assume most if not all of you use phpstorm and it has its own formatter?